dashel/dashel-win32.cpp

```diff
@@ -1379,7 +1447,7 @@ namespace Dashel
 		
 		// Loop in order to consume all events, mostly within lock, excepted for wait
 		lock();
-		do
+		while (true)
```

Comment:
Is there a reason to change the `do { ... } while(true)` into a `while (true) { ... };` excepted aesthetic? If so, I suggest to cancel this change to make the patch as readable as possible.